src/client/src/components/NavBar.vue

```diff
@@ -30,9 +30,9 @@
 						Cases
 					</b-nav-item>
 					<b-nav-item-dropdown class="navBar-font" text="Analytics">
-						<b-dropdown-item href>Analytic 1</b-dropdown-item>
-						<b-dropdown-item href>Analytic 2</b-dropdown-item>
-						<b-dropdown-item href>Analytic 3</b-dropdown-item>
+						<b-dropdown-item to="CompByGender">Compare by Gender</b-dropdown-item>
```

Comment:
the properties to should be set equal to the path, for example, to="/CompByGender"